index.js

```diff
@@ -282,6 +286,10 @@ const registerEvents = () => {
 };
 
 export const config = plugin.config = {
+	validateOnlyWhenConfigured: {
```

Comment:
Can't really think of anything else, but I guess we can keep this and rather explain what it does using a `title` prop. See the other option for example.